addons/hr_expense/models/hr_expense_sheet.py

```diff
@@ -0,0 +1,538 @@
+# -*- coding: utf-8 -*-
```

Comment:
for the review, can you put all of that into the other file. You will add the move to a separated file later